CMakeLists.txt

```diff
@@ -408,13 +408,26 @@ if(IREE_ENABLE_THREADING)
   add_subdirectory(third_party/benchmark EXCLUDE_FROM_ALL)
   iree_set_cpuinfo_cmake_options()
   add_subdirectory(third_party/cpuinfo EXCLUDE_FROM_ALL)
+  # We don't want to install cpuinfo headers.
+  install(
+    TARGETS cpuinfo
+    COMPONENT IREERuntime
+    PUBLIC_HEADER OPTIONAL EXCLUDE_FROM_ALL
+  )
 endif()
 
 add_subdirectory(build_tools/third_party/flatcc EXCLUDE_FROM_ALL)
 
 add_subdirectory(third_party/vulkan_headers EXCLUDE_FROM_ALL)
 
+# Install targets for external libraries that are EXCLUDE_FROM_ALL.
 # TODO(scotttodd): Iterate some more and find a better place for this.
```

Comment:
Maybe remove this TODO, if there's more flatcc configuration going here. I don't really remember what I had in mind for the `iree-flatcc-cli` install / crosscompiling logic there...